### DISCUSSION TITLE
Streaming 2 Separate Audio and Video Files with AntMedia

### DISCUSSION THREAD

**Discussion by hyalicom** [ant-media/Ant-Media-Server#5255] [Plugins/Custom Extensions]
Hi, I have a Live Streaming URL which is in RTMS format! what i want to do is Streaming this video with another audio.. In Short i want to eliminate voice from the Video, and want to replace with my other Audio File, which will be played in the Loop! 

Is there any extension for this?

**Comment by USAMAWIZARD** [comment]
This answer is taken from https://antmedia.io/discover-the-filter-plugin/ Credits : @burak-58 

Merging Video and Audio of Different Streams:
You can merge the video of a stream with the audio of another stream. So you may add any audio to your stream. Please note that you may use this after v2.4.4 version of Ant Media Server.

1.Stream both of the videos.


curl command:

curl -i -X POST -H "Accept: Application/json" -H "Content-Type: application/json" "http://localhost:5080/WebRTCAppEE/rest/v2/filters/create" -d '{"filterId":"addAudio","inputStreams":["stream1","stream2"],"outputStreams":["test"],"videoFilter":"[in0]copy[out0]","audioFilter":"[in1]acopy[out0]","videoEnabled":"true","audioEnabled":"true"}'
Input streams: stream1, stream2

Output streams: test

Video Filter Text:
[in0]copy[out0]

Audio Filter Text:
[in1]acopy[out0]

Explanation: We merge stream1‘s video with the stream2‘s audio into a new stream test.

**Comment by hyalicom** [comment]
Can i please ask anyone, to install antmedia on digital ocean and do this for me, I am not familiar with coding, i can use the ant media, but doesn't know where i have to put these codes and lines! and i am using ant media basic, doesn't have enterprise edition

**Reply by USAMAWIZARD** [reply]
@hyalicom  This feature is only available in Enterprise Edition.You can also have a free trial of Enterprise edition for 14 day before purchasing it please send us a mail to contact@antmedia.io . also were you able to check my answer on [this post ](https://github.com/orgs/ant-media/discussions/5175#discussioncomment-6088708) for this you need to connect to ssh terminal on your cloud provider's dashboard and do just paste the commands.If you need any help we can guide you on the call
